src/components/app/BottomBox.css

```diff
@@ -2,48 +2,86 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-.bottom-box {
+.bottom-box-pane {
   display: flex;
-  height: 100%;
-  flex-flow: column;
+  height: 100%; /* direct child of SplitterLayout */
+  flex-flow: column nowrap;
 }
 
-.bottom-main {
-  position: relative;
+.bottom-sourceview-wrapper,
+.bottom-assemblyview-wrapper {
+  position: relative; /* containing block for status overlay */
   display: flex;
   min-height: 0;
   flex: 1;
   flex-flow: column;
   background: var(--grey-20);
 }
 
+.bottom-box .layout-splitter {
+  position: relative; /* containing block for absolute ::before */
+  width: 1px;
+  border: none;
+  background-color: var(--grey-30) !important;
+}
+
+/* Provide 3px extra grabbable surface on each side of the splitter */
```

Comment:
I noticed that too and filed https://github.com/zesik/react-splitter-layout/issues/67 about it, but the repo seems somewhat unmaintained. It also happens with a regular wide splitter, it's not because of the negative margins, from what I can tell.

Reply:
I see, thanks for filing! Yeah, last commit is from 2019. Doesn't look like it's maintained